spec/index.bs

```diff
@@ -267,6 +289,8 @@ interface GPU {
 };
 </script>
 
+A {{GPU}} object is the entry point to the WebGPU API. It is used to create [=adapters=].
```

Comment:
nit: I'd move this above the IDL, and make this the definition with `<dfn interface>GPU</dfn>`